browser/src/GenePage/GeneTranscriptsTrack.tsx

```diff
@@ -129,15 +140,36 @@ const GeneTranscriptsTrack = ({
         renderTranscriptRightPanel={
           !isTissueExpressionAvailable
             ? null
-            : ({ transcript, width }: any) => {
+            : ({
+                transcript,
+                width,
+              }: {
+                transcript: TranscriptWithTissueExpression
+                width: number
+              }) => {
                 if (width < 36) {
                   return null
                 }
 
-                const meanExpression = mean(Object.values(transcript.gtex_tissue_expression))
-                const maxExpression = max(Object.values(transcript.gtex_tissue_expression))
-                const tissueMostExpressedIn = Object.keys(transcript.gtex_tissue_expression).find(
-                  (tissue: any) => transcript.gtex_tissue_expression[tissue] === maxExpression
+                const meanExpression = mean(
+                  transcript.gtex_tissue_expression.map(
+                    (tissueExpression) => tissueExpression.value
+                  )
+                )!
+                const maxExpression = max(
+                  transcript.gtex_tissue_expression.map(
+                    (tissueExpression) => tissueExpression.value
+                  )
+                )!
+                const tissueMostExpressedIn = transcript.gtex_tissue_expression.find(
```

Comment:
Since `.find(...)` might return undefined if no matching item is found it could helpful to consider checking if `transcript` exists before accessing properties